test/Spago/Build/Monorepo.purs

```diff
@@ -142,8 +142,8 @@ spec = Spec.describe "monorepo" do
       FS.copyTree { src: fixture "monorepo/strict-true-uncensored-warnings", dst: "." }
       let
         errs =
-          [ "[1/2 UnusedName] " <> escapePathInErrMsg [ "package-b", "src", "Main.purs:6:13" ]
-          , "[2/2 UnusedName] " <> escapePathInErrMsg [ "package-b", "test", "Main.purs:6:13" ]
+          [ "[ERROR 1/2 UnusedName] " <> escapePathInErrMsg [ "package-b", "src", "Main.purs:6:13" ]
+          , "[ERROR 2/2 UnusedName] " <> escapePathInErrMsg [ "package-b", "test", "Main.purs:6:13" ]
```

Comment:
Not here: in strict mode warnings are presented as errors. Is that ok? Not sure.

Reply:
Yeah it's good - this is sort of the point of the strict mode.